lib/linguist/heuristics.rb

```diff
@@ -4,7 +4,6 @@ module Linguist
   # A collection of simple heuristics that can be used to better analyze languages.
   class Heuristics
     HEURISTICS_CONSIDER_BYTES = 50 * 1024
-
```

Comment:
I like this empty line 😁. Can we have it back please.

Reply:
done